src/output/outputMod.F90

```diff
@@ -1881,25 +1884,42 @@ subroutine storeSurfsolInBuffer(sps, buffer, nn, blockID,   &
        ! only present in the owned faces, the values of the halo's
        ! are set equal to the nearest physical face. Therefore the
        ! working indices are ii and jj.
-
        do j=rangeFace(2,1), rangeFace(2,2)
-          if(j == rangeFace(2,1)) then
-             jj = min(j + offVis, rangeFace(2,2))
-          else if(j == rangeFace(2,2)) then
-             jj = max(j - offVis, rangeFace(2, 1))
-          else
-             jj = j
-          endif
 
-          do i=rangeFace(1,1), rangeFace(1,2)
-             if(i == rangeFace(1,1)) then
-                ii = min(i + offVis, rangeFace(1,2))
-             else if(i == rangeFace(1,2)) then
-                ii = max(i - offVis, rangeFace(1,1))
-             else
-                ii = i
-             endif
+         !  if satements are used to copy the value of of the interior
+         ! cell since the value isn't defined in the rind ell 
 
+         if (present(jBeg) .and. present(jEnd) .and. (useRindLayer)) then 
```

Comment:
Why did you define i/j beg/end as optional input variables here? Is this routine used anywhere else in adflow?